src/components/locale-redirect.tsx

```diff
@@ -49,25 +63,53 @@ const LocaleRedirectPage = (props: Props) => {
   }, []);
 
   return (
-    <html lang="en">
-      <head>
-        {/* FB process is not completing the locale redirect so the code needs to be here */}
-        <meta name="facebook-domain-verification" content={FB_PIXEL_CODE} />
-      </head>
-      <noscript>
-        <meta
-          httpEquiv="refresh"
-          content={`3;url=${redirectURL}`}
-          data-react-helmet="false"
-        />
-      </noscript>
-      <body>
```

Comment:
not sure i understand why the body and html tags are being removed?

Reply:
oh sorry, forgot to explain that. It turned out that what was in this component as originally written wasn't actually being used as the entire page as I think was intended, but rather it was all going into the body and there was a totally different head to the page that was being generated by default. So after seeing that happening I just removed all of that and everything here goes into the body and the Helmet takes care of the head.

Reply:
unrelated to this PR but i believe the auto-localization (/es if the user's machine/browser is set to spanish) is currently broken for the orgsite--maybe this change will fix it, since it's moving the redirect info into the actual head component...

Reply:
oh good to know, thanks. I'll see if this helps and look into it more otherwise